cli/src/utils/npm.ts

```diff
@@ -10,9 +10,12 @@ export namespace NPM {
     workDir: string,
     pkg?: string,
     exact = true,
-    npm_config_loglevel = "error"
+    npm_config_loglevel = "error",
+    devDependency = false
   ) {
-    const args = ["install"];
+    let args = ["install"];
+
+    if (devDependency) args.push("-D");
```

Comment:
please use the long args, `--save-dev`, it's easier to reason about in the future